### DISCUSSION TITLE
Adding rule to local_rules.xml stops event processing

### DISCUSSION THREAD

**Discussion by chergett** [Security-Onion-Solutions/securityonion#7277] [Unsupported Versions] [answered]
I've added quite a few rules to local_rules.xml to tune for my environment without issue.  I'm having trouble with the following rule, though.  When I add it and restart wazuh, I don't get any more alerts and the event processing seems to come to a halt.  This is the only rule that I've had issues with so far.  Any tips?

`<rule id="60122" level="5" noalert="1" overwrite="yes">
  <if>sid>60105</if_sid>
  <field name="win.system.eventID">^529$|^4625$</field>
  <description>Logon Failure - Unknown user or bad password</description>
  <options>no_full_log</options>
<group>win_authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,gpg13_7.1,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
</rule>`

**Comment by bretthanus** [comment, marked as answer]
It's a stab, but check the syntax in that rule, removing the comma after tsc_cc7.3, preceding the `</group> `

And there doesn't seem to be a `<rule>`in the beginning to match `</rule>` at the end- maybe it wasn't included in the copy/paste.

b

**Reply by chergett** [reply]
There was a typo in the <if_sid> opening tag. DOH!  Thanks for the response.
